main/src/cgeo/geocaching/maps/CachesOverlay.java

```diff
@@ -170,7 +170,7 @@ private int calculateDrawingRadius(final MapProjectionImpl projection) {
         final GeoPointImpl itemGeo = mapItemFactory.getGeoPointBase(itemCoord);
 
         final Geopoint leftCoords = new Geopoint(itemCoord.getLatitude(),
-                itemCoord.getLongitude() - 161 / longitudeLineDistance);
+                itemCoord.getLongitude() - 160.9344f / longitudeLineDistance);
```

Comment:
Instead of using this constant, I would prefer that you use `528 * FEET_TO_KILOMETER * 1000` (constant coming from `IConversion`, that will be made `static final` in a few minutes but that you can use right away).